docs/tenancy/1.x/hooks-database.md

```diff
@@ -103,6 +105,63 @@ class ConfigureTenantDatabase
 }
 ```
 
+> Note: The `Tenancy\Hooks\Database\Events\Drivers\Configuring` event is only used to generate the
+statements required to manage the Tenant's database and user. It is *NOT* used to connect to the
+database that these statements will run on.
+
+#### Remote Database Servers
+In many production applications, the database will be on a seperate server than the application
```

Comment:
This is not what this feature is for. It's for when either multiple web servers connect to the db server or when you don't know which machine would be used

Reply:
I think this is a simple phrasing issue. Merging in order to have the docs, and will follow up w/ you regarding the phrasing.